site/content/en/docs/Guides/Client SDKs/_index.md

```diff
@@ -230,93 +230,115 @@ observable through the Kubernetes API.
 
 ### Counters And Lists
 
-{{< alpha title="Counters And Lists" gate="CountersAndLists" >}}
+{{< alpha title="Counters And Lists" gate="CountsAndLists" >}}
 
 The `Counters` and `Lists` features in the SDK offer a flexible configuration for tracking various entities like 
 players, rooms, and sessions.
 
-{{< alert title="Note" color="info">}}
+Declared keys and default values for Counters and Lists are specified in
+[`GameServer.Spec.Counters` and `GameServer.Spec.Lists`][gameserverspec] respectively.
+
+Modified Counter and List values and capacities will be updated
+in [`GameServer.Status.Counters` and `GameServer.Status.Lists`][gameserverstatus] respectively.
+
+{{% alert title="Note" color="info" %}}
 The SDK batches mutation operations every 1 second for performance reasons. However, changes made and subsequently 
 retrieved through the SDK will be atomically accurate through the SDK, as those values are tracked within the 
 SDK Server sidecar process.
 
-However, changes made through Allocation or the Kubernetes API to `GameServer` List and Counter values will be 
-eventually consistent when being retrieved through the SDK.
+Changes made through Allocation or the Kubernetes API to
+[`GameServer.Spec.Counters` and `GameServer.Spec.Lists`]({{< ref "/docs/Reference/agones_crd_api_reference.html#agones.dev/v1.GameServerSpec" >}})
+will be eventually consistent when being retrieved through the SDK.
 
-Since the SDK update operations back to the `GameServer.status` values is batched asynchronous, it is worth noting that
-any value incremented past a counter or list capacity may be silently truncated to the currently set capacity if 
-modified concurrently through the SDK and Allocation/Kubernetes API.
-{{< /alert >}}
+In the event where a Counter or List may extend past the maximum capacity Allocation/API changes to
+[`GameServer.status`]({{< ref "/docs/Reference/agones_crd_api_reference.html#agones.dev/v1.GameServerStatus" >}}) values
+while a batched Counter or List modification operation has yet to be applied, those values will be silently truncated to
+the capacity of that Counter or List.
```

Comment:
This is still confusing. Subject -> Verb -> Object is an easier format to understand. 

```suggestion
The Agones SDK server batches the update operations of [`GameServer.Status.Counters` and `GameServer.Status.Lists`][gameserverstatus] asynchronously. This means that if you update [`GameServer.status`]({{< ref "/docs/Reference/agones_crd_api_reference.html#agones.dev/v1.GameServerStatus" >}}) values through both the SDK and the Allocation/Kubernetes API, the batch processing may silently truncate some of those values to the capacity of that Counter or List.
```

Reply:
Thank you! That does read much better. Adding change and updating!